desktop/main.ts

```diff
@@ -128,11 +129,12 @@ const quitAndInstallWithUpdate = () => {
 };
 
 /** Menu Item callback to triggers an update check */
```

Comment:
Mind fixing this while we're here?

```suggestion
/** Menu Item callback to trigger an update check */
```